Undeclaried identifier for AI_V4MAPPED

Hello,

I have a build failure on git main:

```
 CC      build-i586/sa/sa.o
src/sa/sa.c:72:20: error: use of undeclared identifier 'AI_V4MAPPED'
        hints.ai_flags  = AI_V4MAPPED | AI_ADDRCONFIG | AI_NUMERICHOST;
                          ^
1 error generated.
gmake: *** [Makefile:117: build-i586/sa/sa.o] Error 1
```

```
$ cc -v
OpenBSD clang version 11.1.0
Target: i386-unknown-openbsd6.9
Thread model: posix
InstalledDir: /usr/bin
```

```
$ gmake -v
GNU Make 4.3
Built for i386-unknown-openbsd6.9
```

I build with `gmake CC=clang` on my OpenBSD i386 machine running a snapshot from yesterday.

Maybe this commit broke it?

https://github.com/baresip/re/commit/14cbecfcae19694019f0ac68cfc6daceb33cb016

--- a/src/sa/sa.c
+++ b/src/sa/sa.c
@@ -69,7 +69,7 @@ int sa_addrinfo(const char *addr, struct sa *sa)
 
 	memset(&hints, 0, sizeof(hints));
 	hints.ai_family = AF_UNSPEC;
-	hints.ai_flags  = AI_V4MAPPED | AI_ADDRCONFIG | AI_NUMERICHOST;
+	hints.ai_flags  = AI_ADDRCONFIG | AI_NUMERICHOST;
 
 	if (getaddrinfo(addr, NULL, &hints, &res0))
 		return EADDRNOTAVAIL;

